app/templates/views/unsubscribe-request-reports-summary.html

```diff
@@ -0,0 +1,50 @@
+{% extends "withnav_template.html" %}
+{% from "components/table.html" import field, list_table, row_heading%}
+{% from "components/page-header.html" import page_header %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+
+{% block service_page_title %}
+  Email unsubscribe requests
+{% endblock %}
+
+{% block backLink %}
+  {{ govukBackLink({ "href": url_for('main.service_dashboard', service_id=current_service.id) }) }}
+{% endblock %}
+
+{% block maincolumn_content %}
+
+    {{ page_header('Email unsubscribe requests') }}
+
+<div class="dashboard-table">
+    {% call(item, row_number) list_table(
+        data,
+        caption="Unsubscribe Request Reports",
+        caption_visible=False,
+        empty_message='If you have unsubscribed requests they will be listed here',
+        field_headings=['Report'],
```

Comment:
Because the table has 2 columns now it should also have 2 field headings
```suggestion
        field_headings=['Report', 'Status'],
```